src/config.js

```diff
@@ -62,7 +62,8 @@ class Dirs {
     d = d || path.join(this.home, category === 'data' ? '.local/share' : '.' + category)
     if (this.windows) d = process.env.LOCALAPPDATA || d
     d = process.env.XDG_DATA_HOME || d
-    d = path.join(d, this._config.name)
+    let n = this._config.name === 'heroku-cli' ? 'heroku' : this._config.name
```

Comment:
@RasPhilCo falling back to package name sounds alright, one thing we should do before releasing this to other groups is beef up the release validation a little bit.  I was thinking about how if there were no fallback then we should fail the release and it does not appear that we do that now if they do not include required attributes in their `cli-engine` https://github.com/heroku/cli-engine/blob/master/bin/cli-release#L36